integration/send-files/e2e/express.spec.ts

```diff
@@ -68,4 +69,44 @@ describe('Express FileSend', () => {
   it('should return an error if the file does not exist', async () => {
     return request(app.getHttpServer()).get('/file/not/exist').expect(400);
   });
+  it('should allow for the client to end the response and be able to make another', async () => {
+    await app.listen(0);
+    const url = await app.getUrl();
+    const [protocol, host, port] = url.replace('[::1]', 'localhost').split(':');
+    const httpOptions = {
+      host: host.replace('//', ''),
+      protocol: `${protocol}:`,
+      port,
+      path: '/file/slow',
+      method: 'GET',
+    };
+    await new Promise<void>(resolve => {
+      const req = http.request(httpOptions, res => {
+        res.on('data', () => {
+          req.destroy();
+        });
+        /* no op */
+        res.on('close', resolve);
+      });
+      req.end();
+    });
```

Comment:
this block of code could also be extracted into a function to give it a meaningful name since it's not trivial. What do you think of:
```suggestion
    await sendRequestAndCancel(httpOptions)
```